zbus/src/connection/connect/win32.rs

```diff
@@ -0,0 +1,41 @@
+#![cfg(target_os = "windows")]
+
+use super::BoxedSplit;
+use crate::{
+    address::{transport::Transport, Address},
+    win32::autolaunch_bus_address,
+    Error, OwnedGuid, Result,
+};
+
+pub(crate) async fn connect(
+    l: &crate::address::transport::Autolaunch<'_>,
```

Comment:
I'm sure you can come up with a better name than `l`.

Reply:
autol ?

Reply:
`autolaunch` isn't super long. :laughing: